worldengine/a_star.py

```diff
@@ -36,42 +34,45 @@ def __eq__( self, n ):
 
 
 class AStar:
+    def __init__(self, map_handler):
+        self.mh = map_handler
+        self.o = []
```

Comment:
maybe these fields names need a comment or to be renamed, maybe they are super clear to people knowing the A* algorithm